sdk/highlight-go/highlight.go

```diff
@@ -243,3 +244,19 @@ func shutdown() {
 	}
 	state = stopped
 }
+
+func SetupGormTracing(ctx context.Context, db *gorm.DB, attrs []attribute.KeyValue) error {
```

Comment:
@Vadman97 is there a convention for naming things like this? Curious what you think the DX should be for setting up the GORM plugin.

Reply:
what if we move to the following 

```go
	highlight.Start(
		highlight.WithServiceName(serviceName),
		highlight.WithServiceVersion(os.Getenv("REACT_APP_COMMIT_SHA")),
                 highlight.WithGORMTracing(db, attrs...)
	)
```
with this method being
```
func WithGORMTracing(db *gorm.DB, attrs attribute.KeyValue...) Option {
...
}
```

looking at gorms docs, i think they prefer `GORM` to `Gorm`?